### ISSUE TITLE
Assets that minis can walk through

### ISSUE THREAD

**Issue by Clickeddaisy** [Bouncyrock/TaleSpire-Beta-AssetRequests#165]
**Describe the Asset**
*What does it look like, how is it used. Paint us a figurative picture.*

Tall grass and wheat that does not have "mass" to them so minis can stand in and walk through them. Maybe give them some "physics" by having them move a bit when a mini walks through and perhaps having them sway in the wind. Also make existing assets like ferns and the like also have the same properties.

**Examples (images, videos, something which gets the point across)**
*Or show us a literal picture. That works too.*

![depositphotos_12164659-stock-photo-woman-in-a-wheat-field](https://user-images.githubusercontent.com/78590813/112147289-33565680-8be5-11eb-8b08-bcef64f1acfa.jpg)

**Additional context**
*Unburden yourself of thoughts and musings here.*

The fields currently in game only work if it is after harvest season in the worlds time so it looks weird otherwise. Also having encounters centered around visibility in those fields would be cool.

<br>**ADD :thumbsup:[thumbs up reaction] to give this request more attention.**

**Comment by xspartanzx** [comment]
Yes! This could work well for tall grass, seaweed forests, fog clouds, bog gas, swarms of insects, etc. Anything that can be seen on the map but doesn't prevent miniatures from moving through the asset. Ideally it would also block vision too when the Fog of War system is revamped
